### DISCUSSION TITLE
0.42.2 beta

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#179] [Announcements]
had some troubles with last-minute bugs on 0.42, i'm sorry.
I think with major changes (like this) I'm going to first publish betas.
PLEASE let me know your feedback on this release, even just "it works" (so far). It's really valuable to me.

[hfs-windows.zip](https://github.com/rejetto/hfs/files/11077166/hfs-windows.zip)

- [new "file menu" thing](https://github.com/rejetto/hfs/wiki/Access-file-menu-option)
- for each folder you can decide what type of files to "accept"
- [method to force login dialog](https://github.com/rejetto/hfs#hidden-features)
- admin/options: limit what IPs can access admin-panel
- admin/accounts: from csv

**Comment by alivesoul88** [comment]
Hi!, excellent application, is there an option to disable password change for users?

**Reply by rejetto** [reply]
Thanks! not yet, but i'll try to have it soon.
Let's keep this discussion about the new beta.
For any further request like this click "issues" above, and then "new issue".

**Reply by alivesoul88** [reply]
Thanks! I keep it in mind for the future

**Reply by gogi100** [reply]
my dialog login is back the header to english translate 'login' instead 'Пријава'
![dialog login](https://user-images.githubusercontent.com/21057856/228017370-7973a1db-9d4c-40ac-b44c-b14788f2c413.jpg)

**Reply by rejetto** [reply]
can't understand what you mean. Try expressing it in your language, then I'll use google translate.

**Reply by gogi100** [reply]
when I log in via my first page which contains the link /login#LOGIN its opens dialog login like on picture above, where is header 'login' instead 'Пријава'

**Reply by rejetto** [reply]
i've managed to fix it

**Reply by rejetto** [reply]
@alivesoul88  feature is finally available in version 0.49

**Comment by gogi100** [comment]
one more suggestion, when on page in the hfs i make log off can you set that redirecting on the log on dialog window. i set short video

https://user-images.githubusercontent.com/21057856/228020460-0e34d55c-dad6-4d9c-bff7-83960b37e9a7.mov

**Reply by rejetto** [reply]
after logout I need to refresh the list, because it doesn't reflect the state anymore.
If the folder is protected, you'll get the login dialog, so this should be good for your case.

**Comment by rejetto** [comment]
[beta2](https://github.com/rejetto/hfs/files/11083353/hfs-windows.zip)

this new beta includes 3 more fixes

**Comment by harrischrisa** [comment]
I just installed beta 2. It certainly starts up OK and works but I haven't had an opportunity to test everything

Chris
